src/tests/components/__snapshots__/SearchForm.test.js.snap

```diff
@@ -0,0 +1,8 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`SearchForm renders the input field and button 1`] = `
```

Comment:
This snapshot doesnt make much sense... In order to get this to work properly you need to change how youve written the test a bit. The test test file for details